.github/workflows/maven.yml

```diff
@@ -109,11 +106,9 @@ jobs:
     # Install dependencies
     - name: Install ci.ant and ci.common
       run: |
-        cd C:/ci.ant
-        mvn clean install --batch-mode --no-transfer-progress --errors -DtrimStackTrace=false
-        cd C:/ci.common
-        mvn clean install --batch-mode --no-transfer-progress --errors -DtrimStackTrace=false
+        .\mvnw.cmd -V clean install -f ..\ci.ant --batch-mode --no-transfer-progress --errors -DtrimStackTrace=false
```

Comment:
Can we just go back to the separate lines with `cd` and `.\mvnw.cmd`? The `C:` drive notation is important.